indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -439,6 +439,13 @@ public void handle()
             );
             return;
           }
+          if (SupervisorStateManager.BasicState.IDLE == getState()) {
+            log.info(
```

Comment:
How often does this get logged? Seems like once a minute by default? Should we lower to debug?